test/test_regression_2.py

```diff
@@ -0,0 +1,54 @@
+"""
+test_regression_1.py and test_regression_2.py together
+test:
+- running some numbers through two versions of sru, checking they come out the sam
+- save sru in older version, and loading in new version
+
+these scripts are adapted from those by Sean Adler
+"""
+import torch
+import sru
+import argparse
+
+
+EPSILON = 1e-6
+
+
+def run(args):
+    torch.manual_seed(2)  # so the model is initialized differently than first stage
+
+    input_size = 300
+    hidden_size = 250
+
+    model = sru.SRU(
+        input_size,
+        hidden_size,
+        num_layers=4,
+        bidirectional=True,
+        dropout=0.1,
+        rescale=False
+    ).eval()
+
+    output_sru_235 = torch.load(args.in_outputs)
+    model.load_state_dict(torch.load(args.in_model))
+    inputs = torch.load(args.in_inputs)
+
+    with torch.no_grad():
+        output_sru_250 = model(inputs)
```

Comment:
Probably a huge nitpick, but this doesn't always test version 2.5.0 right? Maybe `output_sru_base` and `output_sru_current` would match your bash vars better?